client/lib/formatting/decode-entities.ts

```diff
@@ -1,6 +1,6 @@
-import decode from './decode';
```

Comment:
This is the change that inflates the `entry-*` bundles. By requiring the `node` version, we are shipping a large `he` package for decoding HTML entities. Instead of a browser implementation that has just a few lines and uses DOM APIs.

Reply:
You probably saved me hours with this comment. Many many thanks!